packages/frontend-2/.env.example

```diff
@@ -19,4 +19,10 @@ NUXT_PUBLIC_SERVER_NAME=local
 NUXT_REDIS_URL=redis://localhost:6379
 
 # Set to true for extra logging related to viewer
-NUXT_PUBLIC_VIEWER_DEBUG=false
\ No newline at end of file
+NUXT_PUBLIC_VIEWER_DEBUG=false
+
+# RUM monitoring
+NUXT_PUBLIC_RAYGUN_KEY=
```

Comment:
@iainsproat @gjedlicska I'd like your assistance in setting these up **only** for latest.speckle.systems